assets/src/apps/v2/bus_eink.tsx

```diff
@@ -96,6 +96,13 @@ const App = (): JSX.Element => {
   return (
     <Router>
       <Switch>
+        <Route exact path={["/v2/screen/:id", "/v2/screen/pending/:id"]}>
```

Comment:
(I moved the main route to the top for visibility. Looks like it was at the bottom only because it was matching paths like `/v2/screen/:id/simulation`; adding `exact` fixes that.)